java/test/jmri/jmrix/loconet/LnNetworkPortControllerTest.java

```diff
@@ -12,7 +14,13 @@
 public class LnNetworkPortControllerTest extends jmri.jmrix.AbstractNetworkPortControllerTestBase {
 
     private LocoNetSystemConnectionMemo memo;
- 
+
+    @Test
+    public void testGetAndSetOutputInterval() {
```

Comment:
Shouldn’t this be moved up to the parent test class?  Also, why test this here, since the value is being set on the memo, not the port controller?

Reply:
As before, took out this test here